samples/MusicStore.Spa/Infrastructure/SmartJsonResult.cs

```diff
@@ -29,8 +28,8 @@ public override Task ExecuteResultAsync(ActionContext context)
             //    context.HttpContext.Response.ContentType = "application/json";
             //    context.HttpContext.Response.ContentEncoding = Encoding.UTF8;
             //}
-            
-            return context.HttpContext.Response.WriteAsync(JsonConvert.SerializeObject(Data, Settings ?? new JsonSerializerSettings()));
+            var settings = Settings ?? JsonSerializerSettingsProvider.CreateSerializerSettings();
```

Comment:
We shouldn't see `new JsonSerializerSettings()` outside MVC unit tests and maybe not there.